.github/workflows/docs-link-check.yaml

```diff
@@ -0,0 +1,193 @@
+            
+            # Format cross-reference results
+            XREF_RESULTS="## Broken Cross-References\n\n"
+            XREF_RESULTS+="The following cross-references were broken in the recent merge:\n\n"
+            XREF_RESULTS+="${{ steps.cross-references.outputs.cross_ref_results }}"
+            
+            ISSUE_CONTENT+="$XREF_RESULTS\n\n"
+          fi
+          
+          # Add guidance if issues were found
+          if [ "$HAS_ISSUES" == "true" ]; then
+            ISSUE_CONTENT+="## How to Fix\n\n"
+            ISSUE_CONTENT+="### For Broken Links\n\n"
+            ISSUE_CONTENT+="1. Check if the link is correct\n"
+            ISSUE_CONTENT+="2. Update the link if needed or remove it\n"
+            ISSUE_CONTENT+="3. If the link is valid but fails the check, consider adding it to the ignore list in `.github/docs/.lycheeignore`\n\n"
+            
+            # Use cat with heredoc instead of echo -e for more reliable newline handling
+            cat << 'EOT' >> /tmp/issue-guidance.txt
+            ### Additional Guidance
+            - For 404 errors: Check if the resource was moved or renamed
+            - For timeout errors: The site might be temporarily down, consider retrying
+            - For redirect issues: Update to the final URL
+            EOT
+            ISSUE_CONTENT+=$(cat /tmp/issue-guidance.txt)
+            
+            ISSUE_CONTENT+="### For Broken Cross-References\n\n"
+            ISSUE_CONTENT+="1. Update references to deleted or renamed files\n"
+            ISSUE_CONTENT+="2. Update references to removed headings\n"
+            ISSUE_CONTENT+="3. Check for references in both documentation and code\n\n"
+            
+            ISSUE_CONTENT+="---\n\nThis issue was automatically created by the Documentation Post-Merge Checks workflow."
+            
+            # Save for issue creation
+            echo "issue_title<<EOF" >> $GITHUB_OUTPUT
+            echo "$ISSUE_TITLE" >> $GITHUB_OUTPUT
+            echo "EOF" >> $GITHUB_OUTPUT
+            
+            echo "issue_content<<EOF" >> $GITHUB_OUTPUT
+            echo -e "$ISSUE_CONTENT" >> $GITHUB_OUTPUT
+            echo "EOF" >> $GITHUB_OUTPUT
+            
+            echo "has_issues=true" >> $GITHUB_OUTPUT
+          else
+            echo "No issues found in documentation checks"
+            echo "has_issues=false" >> $GITHUB_OUTPUT
+          fi
+
+      # Create a single GitHub issue for all problems found
+      - name: Create GitHub issue
+        if: steps.process-results.outputs.has_issues == 'true'
+        id: create-issue
+        uses: peter-evans/create-issue-from-file@v4.0.1
+        with:
+          title: ${{ steps.process-results.outputs.issue_title }}
+          content: ${{ steps.process-results.outputs.issue_content }}
+          labels: |
+            documentation
+            bug
+            needs-triage
+          # Configure assignee through CODEOWNERS or a team instead of hardcoding
```

Comment:
I don't think it's a good idea to create issues from a workflow. They can grow pretty quickly, and we also have flakes.